src/track.ts

```diff
@@ -352,9 +352,9 @@ function isDebugMode(): boolean {
   if (account?.user.email.endsWith("@google.com")) {
     try {
       require("../tsconfig.json");
-      logger.debug(
-        `Using Google Analytics in DEBUG mode. Emulators (+ UI) events will be shown in GA Debug View only.`,
-      );
+      // logger.debug(
```

Comment:
Uncomment this? It only shows up for googlers anyway

Reply:
Ah yeah this was back when I was trying to do MCP as a normal command and the logs going to stdout were blowing things up. Uncommented.